app/representers/cms/file_representer.rb

```diff
@@ -1,18 +1,22 @@
-module CMS::FileRepresenter
-  include ThreeScale::JSONRepresenter
+# frozen_string_literal: true
 
-  wraps_resource
+module CMS
+  module FileRepresenter
+    include ThreeScale::JSONRepresenter
 
-  property :id
-  property :created_at
-  property :updated_at
-  property :title
-  property :path
-  property :url
-  property :section_id
-  property :tag_list
+    wraps_resource
```

Comment:
I think you should use the same formula in all resources:

`wraps_resource ->(*) { self.class.data_tag }`

So the given `data_tag` is always used

Reply:
This is only needed for the cases where the tag may change. Like for `section` and `builtin_section` that share the same representer.
For this case, data will always be wrapped by `file`.

Reply:
Yes but it's wrapped by `file` automatically probably because that's the class name, I don't know where that `file` comes from. The point of the `data_tag` thing is for us to explicitly define the tag names in one place and have it used for JSON and XML formats, if we decide to modify `data_tag` in the future, it will have effect on the XML but not in the JSON.

Reply:
Indeed it comes from the class name.
Added the explicit data tag wrap in e3d778b.